dashboard/app/controllers/application_controller.rb

```diff
@@ -8,9 +8,7 @@ class ApplicationController < ActionController::Base
   include LocaleHelper
   include ApplicationHelper
 
-  # Commenting this stuff out because even if we don't have a reader configured
-  # it will set stuff in the session.
-  # include SeamlessDatabasePool::ControllerFilter
+  include SeamlessDatabasePool::ControllerFilter
   # use_database_pool :all => :master
```

Comment:
Does this need to be uncommented? Or does it default to master?

Reply:
Yes, it defaults to master, and there may be some subtle differences between using the default and setting it explicitly. I left this as-is to reduce the PR diff and to avoid unintentionally changing behavior.